docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -352,19 +359,22 @@ as simple comparison pushdown was implemented first via summary, and more
 complex filters such as ``LIKE`` which required more expressive predicates
 were added later.
 
-If a constraint can only be pushed down partially, e.g. a connector for
+If a constraint can only be pushed down partially, for example a connector for
 a database that doesn't support range matching is used in a query with
 ``WHERE x = 2 AND y > 5``, the ``y`` column constraint should be
 returned in the ``ConstraintApplicationResult`` from ``applyFilter``.
-In this case the ``y > 5`` part of the filter will be applied in Trino,
+In this case the ``y > 5`` part of the filter is applied in Trino,
```

Comment:
In this case, the ``y > 5`` condition is

("part of the filter" is very informal. Please be specific about that part of the filter. It is a condition.